### ISSUE TITLE
STAR G640 tablet and pen not working ingame

### ISSUE THREAD

**Issue by Creedin** [ppy/osu#7326]
**With Raw Input**
STAR G640 Pen not working in osu!lazer when the window is selected. If osu window is not selected, and I put my pen on my tablet, the cursor will move on the screen, but the left click from my pen won't work on the osu window. Also, when I select the window with my mouse, the cursor stops moving again.

**Without Raw Input**
Same issue except when the osu window is not selected, the tablet will track cursor if the pen is hovering over tablet until I tap the tablet. When that happens the cursor stops moving when hovering the pen, but it will jump to where I tap it on the tablet.

**Version**
Lazer 2019.1219.0

**Note**
My tablet and pen works normal on stable with raw input on. I can hover or keep pen on the tablet directly.
Stable mouse settings:
![image](https://user-images.githubusercontent.com/30446608/71336902-c5693e00-250e-11ea-87e4-458933e6b5a5.png)

**Comment by UselessToucan** [comment]
> STAR G640

Which driver do you use? Would be nice to have a version number or a download link.

Also, did you check other window modes(borderless, fullscreen)?

**Comment by Creedin** [comment]
> Which driver do you use? Would be nice to have a version number or a download link.

I use the PenTablet by XP-PEN Ver:1.6.3(Build:159f9682)
![image](https://user-images.githubusercontent.com/30446608/71448587-86500e00-2703-11ea-8f9f-ee4515a071af.png)


> Also, did you check other window modes(borderless, fullscreen)?
    
Yes I tried all modes.

**Comment by revelsix** [comment]
For what it's worth, I have the same exact model of tablet and my game works fine. Have you tried other drivers?
I'm using [hawku TabletDriverV0.2.3,](https://github.com/hawku/TabletDriver/releases) I'll investigate using the drivers you're currently running soon.

**Comment by peppy** [comment]
Removed reference/link to untrusted software. The game should not require unofficial drivers to work.

**Comment by revelsix** [comment]
Fair enough point I guess

@Creedin The download site for XP-Pen uses a different numbering system for versions, you may have to look at the .zip file it came in, the number should look something like "20191126" (If using the one made for osu!, it should also have OSU right before the number.)

**Comment by Joehuu** [comment]
I could reproduce with an old driver still listed on the site: `Star G640S, G640, G430S, G430, G540 Pro For osu!`, and the lazer version listed here. But, updating either to latest will fix the issue.

TL;DR: can't reproduce on latest lazer and latest/beta drivers. I suggest updating them as I can't reproduce anymore.

Will close for now.
